edb/graphql-rewrite/src/entry_point.rs

```diff
@@ -145,12 +215,79 @@ pub fn rewrite(operation: Option<&str>, s: &str) -> Result<Entry, Error> {
         }
         oper.ok_or_else(|| Error::NotFound("no operation found".into()))?
     };
-    let (tokens, end_pos) = token_array(s)?;
-    let mut src_tokens = TokenVec::new(tokens);
+    let (all_src_tokens, end_pos) = token_array(s)?;
+    let mut src_tokens = TokenVec::new(&all_src_tokens);
     let mut tokens = Vec::with_capacity(src_tokens.len());
 
     let mut variables = Vec::new();
+    let mut defaults = BTreeMap::new();
+    let mut key_vars = BTreeSet::new();
+    let mut value_positions = HashSet::new();
 
+    visit_directives(&mut key_vars, &mut value_positions, &oper);
+
+    for var in &oper.variable_definitions {
+        if var.name.starts_with("_edb_arg__") {
```

Comment:
Please use `__{..}__` pattern as we do for other similar things -- identifiers like that are reserved for EdgeDB in EdgeQL and everywhere else.

Reply:
GraphQL also has `__xxx` pattern reserved for introspection purposes. So it chokes on that, see https://github.com/edgedb/edgedb/pull/1289